src/components/newOrderButton.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { AppButton } from "./appButton";
+import { useDispatch } from "react-redux";
+import { initialBreadState } from "../store/bread/breadSlice";
+import { initialAddToOrderState } from "../store/addToOrder/addToOrderSlice";
+import { updateCurrentScene } from "../store/appControl/appControlSlice";
+import { initialCheeseState } from "../store/cheese/cheeseSlice";
+import { initialMeatState } from "../store/meat/meatSlice";
+import { initialDresingState } from "../store/dressing/dressingSlice";
+import { initialVegetablesState } from "../store/vegetables/vegetablesSlice";
+import { initialEggState } from "../store/egg/eggSlice";
+import { initialSpreadsState } from "../store/spreads/spreadsSlice";
+import { initialServingState } from "../store/serving/servingSlice";
+import { initialToppingState } from "../store/topping/toppingSlice";
+import { initialNameState } from "../store/name/nameSlice";
+
+export function NewOrderButton() {
+  const dispatch = useDispatch();
+  const handleClickNewOrder = () => {
```

Comment:
you can create one action, for example resetState and add extra reducer to each slice which resets the state tothe initial one. This way you want need to create this massive function

https://redux-toolkit.js.org/api/createSlice#extrareducers